scripts/sync-to-downstream.sh

```
@@ -474,9 +490,438 @@ run_audit() {
+  local branch=$5
+  local targets=$6  # newline-separated paths
+  local kit_list=$7
+  local templated_list=$8
+  local short_sha=${sha:0:7}
+
+  local workspace
+  workspace=$(mktemp -d -t "mergepath-sync-$consumer_name") || {
+    err "could not create workspace tmpdir"
+    return 1
+  }
+  trap "rm -rf '$workspace'" RETURN
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)